modules/cart/src/Plugin/views/field/EmptyCartButton.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace Drupal\commerce_cart\Plugin\views\field;
+
+use Drupal\commerce_cart\CartManagerInterface;
+use Drupal\commerce_order\Entity\Order;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\views\Plugin\views\field\FieldPluginBase;
+use Drupal\views\Plugin\views\field\UncacheableFieldHandlerTrait;
+use Drupal\views\ResultRow;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Defines a form element for emptying the cart.
+ *
+ * @ViewsField("commerce_order_item_empty_cart_button")
+ */
+class EmptyCartButton extends FieldPluginBase {
```

Comment:
Hm. So we're not providing it by default. But, I believe tests are our first line of defense when it comes to documentation. Every feature should have a test. That way we know it works and it exists. Otherwise we have features without coverage and without implementation documentation.

A test module should provide a view which implements this button.